examples/pcb-aoi/singletask_learning_bench/benchmarkingjob.yaml

```diff
@@ -0,0 +1,67 @@
+benchmarkingjob:
+  # job name of bechmarking; string type;
+  name: "benchmarkingjob"
+  # the url address of job workspace that will reserve the output of tests; string type;
+  workspace: "/ianvs/singletask_learning_bench/workspace"
+
+  # the url address of test environment configuration file; string type;
+  # the file format supports yaml/yml;
+  testenv: "./examples/pcb-aoi/singletask_learning_bench/testenv/testenv.yaml"
+
+  # test algorithm configuration files; list type;
+  algorithms:
```

Comment:
It is possible to introduce other test objects here rather than algorithm. For a more extendable .yaml, it is suggested to add an abstraction of test object to hold all possible implementations and ensure singleton implementation of the test object.